assets/js/builder/Schemas/Quiz.js

```diff
@@ -1,8 +1,10 @@
 /**
- * Quiz Schema
+ * Quiz Schema.
  *
- * @since    3.17.6
- * @version  3.24.0
+ * @since 3.17.6
+ * @since [version] Added upsell for Question Bank.
+ *
+ * @version [version]
```

Comment:
```suggestion
 * @since 3.17.6
 * @since [version] Added upsell for Question Bank.
 * @version [version]
```
and you also made `random_question` conditionally visible.